docs/user-guide/advanced-usage.rst

```diff
@@ -59,38 +59,54 @@ You can customize this by passing in a value for ``log_entry``::
   # after configuring `unify.logger`
   unify.log_entry = log_entry
 
-Custom HTTP requests
---------------------
+Custom HTTP requests and Unversioned API Access
+-----------------------------------------------
 
-We encourage you to use the higher-level, object-oriented interface offered by
-the Python Client. If you aren't sure if you need to send low-level HTTP requests,
-you probably don't.
+We encourage you to use the high-level, object-oriented interface offered by
+the Python Client. If you aren't sure whether you need to send low-level HTTP
+requests, you probably don't.
 
-But sometimes it's useful to directly send HTTP requests to Unify.
+But sometimes it's useful to directly send HTTP requests to Unify; for example,
+Unify has many APIs that are not covered by the higher-level interface (most of
+which are neither versioned nor supported). You can still call these endpoints
+using the Python Client, but you'll need to work with raw ``Response`` objects.
 
-Specific endpoint
-^^^^^^^^^^^^^^^^^
+Custom endpoint
+^^^^^^^^^^^^^^^
 
-The client exposes a ``request`` method with the same interface as ``requests.request``::
+The client exposes a ``request`` method with the same interface as
+``requests.request``::
 
   # import Python Client library and configure your client
 
   unify = Client(auth)
   # do stuff with the `unify` client
 
   # now I NEED to send a request to a specific endpoint
-  response = unify.request('GET', 'some/specific/endpoint')
+  response = unify.request('GET', 'relative/path/to/resource')
 
-You can also use the ``get``, ``post``, ``put``, ``delete`` convenience methods::
+This will post a request relative to the base_path registered with the client.
```

Comment:
```suggestion
This will send a request relative to the base_path registered with the client.
```

since "post" could be misconstrued as HTTP POST method